docs/resources/aws_config_delivery_channel.md

```diff
@@ -0,0 +1,77 @@
+---
+title: About the aws_config_delivery_channel Resource
+---
+
+# aws_config_delivery_channel
+
+Use the `aws_config_delivery_channel` InSpec audit resource to test properties of your aws config service.
+
+<br>
+
+## Syntax
+
+An `aws_config_delivery_channel` resource block declares the tests for a single AWS configuration delivery channel.
+
+    describe aws_config_delivery_channel('my_channel') do
+      it { should exist }
+    end
+
+    describe aws_config_delivery_channel(recorder_name: 'my-channel') do
+      it { should exist }
+    end
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Test if the recorder is active and recording.
+
+    describe aws_config_delivery_channel(recorder_name: 'my-recorder') do
+      it { should be_recording }
+    end
+    
+## Properties
+    
+### s3_bucket_name
+
+Provides the name of the s3 bucket that the channel takes snaphots of and logs configuration history.
```

Comment:
The delivery channel *writes* configuration changes to the bucket.  The channel does not snapshot or read the bucket.